cmd/eksctl-anywhere/cmd/installpackagecontroller.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-anywhere/pkg/curatedpackages"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/version"
+)
+
+type installControllerOptions struct {
+	kubeVersion string
+}
+
+var ico = &installControllerOptions{}
+
+func init() {
+	installCmd.AddCommand(installPackageControllerCommand)
+	installPackageControllerCommand.Flags().StringVar(&ico.kubeVersion, "kube-version", "", "Bundle version to use")
+	if err := installPackageControllerCommand.MarkFlagRequired("kube-version"); err != nil {
+		log.Fatalf("Error marking flag as required: %v", err)
+	}
+}
+
+var installPackageControllerCommand = &cobra.Command{
+	Use:          "packagecontroller",
+	Aliases:      []string{"packagecontroller"},
```

Comment:
Isn't this superfluous?

Reply:
What would be a more concise manner you would think?

Reply:
Its not about being concise, its the fact the alias matches its use. Alias' are for _alternate_ ways of calling the command.

From [the docs](https://pkg.go.dev/github.com/spf13/cobra#Command):

> Aliases is an array of aliases that can be used instead of the first word in Use.

Reply:
Gotcha. Thanks